projects/adventure/adv.py

```diff
@@ -28,7 +30,11 @@
 
 
 # FILL THIS IN
-traversalPath = TraverseAll(player)
+time_start = time.time()
+traversalPath = traverse_all(player)
```

Comment:
Great job Stephen ... just importing function here .. and implementing code in another file..